### PR TITLE
Public Transit, Geocoding, Bugfix, Deliveroo API

### PR DESCRIPTION
**Pull request by Fluepke**
Fixes issue #190.

Changes: Added geocoding support using NominatimService.
Geocoding services are yet to be implemented in susi dreams.
Future use would be to automatically convert place names to geo coordinates to feed them into API requests.

### CODE REVIEW

**Comment by Fluepke** [comment]
If necessary, I can also split this up into two PRs.

**Comment by mariobehling** [comment]
@Fluepke Please ask and assign PRs to your co-developers using the @ feature in description. Thanks.

**Comment by Fluepke** [comment]
This pull request has just grown a little more: Now there's the deliveroo API and public transit support.

**Comment by Fluepke** [comment]
@saurabhjn76 @DravitLochan please review

File: src/ai/susi/geo/AbstractGeoPoint.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public static double distance(final double lat1, final double lon1, final double
         return EQUATOR_EARTH_RADIUS * c;
     }
 
-    static final double EQUATOR_EARTH_RADIUS = 63781370.0d;
```
Comment (Fluepke) [inline src/ai/susi/geo/AbstractGeoPoint.java]: Bug #190 fixed here

**Comment by Fluepke** [comment]
Removed .classpath as suggested by @saurabhjn76

**Comment by mariobehling** [comment]
Sorry for not responding earlier, but yes, I think this PR should be split up in different PRs. It seems like you  are covering points outside of issue #190 in this PR. Please open an issue for points outside of this issue. Thank you.

**Comment by Fluepke** [comment]
You can pull in the necessary git submodule via `git submodule update --recursive --remote`

**Pull request merged**